assembl/static2/js/app/pages/discussionAdmin.jsx

```diff
@@ -79,7 +96,13 @@ type Props = {
   deleteTextField: Function,
   profileOptionsHasChanged: boolean,
   refetchTextFields: Function,
-  textFields: string
+  textFields: string,
+  updateDiscussionPhase: Function,
+  createDiscussionPhase: Function,
+  deleteDiscussionPhase: Function,
+  refetchTimeline: Function,
+  phasesHaveChanged: boolean,
+  phases: Array<Object>
```

Comment:
it could be nice to generated flow types/fragments and use it here (but do it after you merged/rebased develop in the branch). Look at `flow/README.md` for how-to

Reply:
✅